.github/workflows/ci_gpu.yml

```diff
@@ -0,0 +1,27 @@
+name: CI GPU
+
+on: push
+jobs:
+  nightly:
+    runs-on: [self-hosted, ci-gpu]
+    steps:
```

Comment:
could you add clean up before build start? eg.
  - name: Clean up previous run
        run: |
          echo "Cleaning up previous run"
          rm -rf "${{ github.workspace }}"

Reply:
This didn't quite work

Error: An error occurred trying to start process '/home/ubuntu/actions-runner/externals/node12/bin/node' with working directory '/home/ubuntu/actions-runner/_work/serve/serve'. No such file or directory